packages/nocodb/src/db/sql-client/lib/pg/PgClient.ts

```diff
@@ -876,9 +876,9 @@ class PGClient extends KnexClient {
         // todo : need to find if column is unique or not
         // column['unique'] = response.rows[i]['cst'].indexOf('UNIQUE') === -1 ? false : true;
 
-        column.cdf = response.rows[i].cdf;
-        // ? response.rows[i].cdf.split("::")[0].replace(/'/g, "")
-        // : response.rows[i].cdf;
+        column.cdf = response.rows[i].cdf
+          ? response.rows[i].cdf.split('::')[0].replace(/'/g, '')
```

Comment:
1. What if value contains `::` ? Do something like `.replace(/::\w+$/, '')`
2. What if value contains `'` ? have to remove only leading and trailing `'`

Reply:
upated